File: seed/csharp-sdk/exhaustive/no-root-namespace-for-core-classes/Snippets/Example38.cs
using SeedApi;
using SeedApi.Core;
using SeedApi.Endpoints;

public partial class Examples
{
    public async Task Example38() {
        var client = new SeedApiClient(
            token: "<token>",
            clientOptions: new ClientOptions {
                BaseUrl = "https://api.fern.com"
            }
        );

        await client.Endpoints.HttpMethods.TestDeleteAsync(
            new TestDeleteHttpMethodsRequest {
                Id = "id"
            }
        );
    }

}

File: seed/java-sdk/exhaustive/signed_publish/src/main/java/com/snippets/Example38.java
package com.snippets;

import com.seed.api.SeedApiClient;
import com.seed.api.resources.endpoints.httpmethods.requests.TestDeleteHttpMethodsRequest;

public class Example38 {
    public static void main(String[] args) {
        SeedApiClient client = SeedApiClient.builder()
                .token("<token>")
                .url("https://api.fern.com")
                .build();

        client.endpoints()
                .httpMethods()
                .testDelete("id", TestDeleteHttpMethodsRequest.builder().build());
    }
}
